.github/workflows/rector-cs.yml

```diff
@@ -9,18 +11,20 @@ on:
       - 'infection.json.dist'
       - 'psalm.xml'
 
-name: rector
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
 
 jobs:
   rector:
-    uses: yiisoft/actions/.github/workflows/rector.yml@master
+    uses: yiisoft/actions/.github/workflows/rector-cs.yml@master
     secrets:
       token: ${{ secrets.YIISOFT_GITHUB_TOKEN }}
     with:
       repository: ${{ github.event.pull_request.head.repo.full_name }}
-      os: >-
-        ['ubuntu-latest']
-      php: >-
-        ['8.4']
+      php: '8.1'
```

Comment:
```suggestion
      php: '8.4'
```

Why not 8.4?

Reply:
PHP CS Fixer recommendation:

`You are running PHP CS Fixer on PHP 8.2.29, but the minimum PHP version supported by your project in composer.json is PHP 8.1. Executing PHP CS Fixer on newer PHP versions may introduce syntax or features not yet available in PHP 8.1, which could cause issues under that version. It is recommended to run PHP CS Fixer on PHP 8.1, to fit your project specifics.`